### PR TITLE
CI: Add Ruby 3.1

### PR DESCRIPTION
**Pull request by olleolleolle**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -13,12 +13,16 @@ jobs:
 
     strategy:
       matrix:
-        ruby-version: ["3.0", "2.7", "2.6"]
+        ruby-version:
+          - 3.1
+          - "3.0"
```
Comment (henrik) [inline .github/workflows/ci.yml:18]:
```suggestion
          - 3.0
```

**Comment by henrik** [comment]
¡Gracias!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (olleolleolle) [reply .github/workflows/ci.yml]: The idea of the quotes is: the YAML float with a zero on it becomes wrongly stringified to "3" and not "3.0", which would mess with interpolation. So, people who care about the output they make with this value, they quote it. There's a GitHub Actions Issue about it.

Comment (henrik) [reply .github/workflows/ci.yml]: Ah, thanks! Fixing.

**Pull request merged**